.github/workflows/build-container.yml

```diff
@@ -13,24 +13,25 @@ jobs:
     name: 'Build amc2moodle container'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```

Comment:
I suggest `@main` instead of `@v4` for all actions.
Since the version doesn't matter, `@main ` allow to use always the last version.

Reply:
Caution @main is not always available (check for @master)